testsuite/features/secondary/min_migrate_to_trad.feature

```diff
@@ -0,0 +1,111 @@
+
+  Scenario: Wait until the Salt minion appears
+    When I wait until onboarding is completed for "sle_client" salt minion
+
+  Scenario: Check that the migrated system is now a minion
+    Given I am on the Systems overview page of this "sle_client"
+    When I follow "Properties" in the content area
+    Then I wait until I see "Base System Type:.*Salt" regex, refreshing the page
+
+  @proxy
+  Scenario: Check connection from migrated minion to proxy
+    Given I am on the Systems overview page of this "sle_client"
+    When I follow "Details" in the content area
+    And I follow "Connection" in the content area
+    Then I should see "proxy" short hostname
+
+  @proxy
+  Scenario: Check registration on proxy of migrated minion
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "Details" in the content area
+    And I follow "Proxy" in the content area
+    Then I should see "sle_client" hostname
+
+  Scenario: Unregister migrated client
+    Given I am on the Systems overview page of this "sle_client"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    Then "sle_client" should not be registered
+
+  @susemanager
+  Scenario: Register minion again as traditional client
+    When I enable client tools repositories on "sle_client"
+    And I install the traditional stack utils on "sle_client"
+    And I remove package "salt-minion" from this "sle_client"
+    And I bootstrap traditional client "sle_client" using bootstrap script with activation key "1-SUSE-KEY-x86_64" from the proxy
+    Then I should see "sle_client" via spacecmd
+
+  @uyuni
+  Scenario: Register minion again as traditional client
+    When I enable client tools repositories on "sle_client"
+    And I install the traditional stack utils on "sle_client"
+    And I remove package "venv-salt-minion" from this "sle_client"
+    And I bootstrap traditional client "sle_client" using bootstrap script with activation key "1-SUSE-KEY-x86_64" from the proxy
+    Then I should see "sle_client" via spacecmd
+
+  Scenario: Wait until the traditional client appears
+    When I wait until onboarding is completed for "sle_client"
+
+  Scenario: Check that the migrated minion is again a traditional client
+    Given I am on the Systems overview page of this "sle_client"
+    When I follow "Properties" in the content area
+    Then I wait until I see "Base System Type:.*Management" regex, refreshing the page
+
+  Scenario: Unregister traditional client
+    Given I am on the Systems overview page of this "sle_client"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    Then "sle_client" should not be registered
+
+  Scenario: Cleanup: remove leftover package of traditional client
```

Comment:
Please add a comment line before this step, where we describe that this is a workaround for a reported bug, that will not be fixed.

Reply:
I left some comment and added the Bugzilla ID.